xml/System.Net.Http.Headers/HttpHeadersNonValidated.xml

```diff
@@ -264,10 +264,10 @@ Multiple header values associated with the same header name are considered to be
         <Parameter Name="value" Type="System.Net.Http.Headers.HeaderStringValues" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">The key to locate.</param>
+        <param name="value">When this method returns, the value associated with the specified key, if the key is found; otherwise, the default value of <see cref="T:System.Net.Http.Headers.HeaderStringValues" />. This parameter is passed uninitialized.</param>
```

Comment:
```suggestion
        <param name="value">When this method returns, contains the value associated with the specified key, if the key is found; otherwise, contains the default value of <see cref="T:System.Net.Http.Headers.HeaderStringValues" />. This parameter is passed uninitialized.</param>
```